controllers/terraform_controller.go

```diff
@@ -923,6 +923,14 @@ terraform {
 
 	log.Info("tfexec initialized terraform")
 
+	workspaceRequest := &runner.WorkspaceRequest{
+		TfInstance: tfInstance,
+		Terraform:  terraformBytes,
+	}
+	workspaceReply, _ := runnerClient.SelectWorkspace(ctx, workspaceRequest)
+
+	log.Info(fmt.Sprintf("workspace select reply: %s", workspaceReply.Message))
+
```

Comment:
I suggest you put a check in place either here or in the `SelectWorkspace` runner function.